src/SYS.ASM

```diff
@@ -426,17 +429,17 @@ command:                                        ; Main loop: Retrieve command fr
     sub ax, input_buffer                        ; Subtract the starting index of the command input buffer
     mov word [pipe_offset], ax                  ; This is our offset to look from next
 
-    jmp .load_file                              ; We are done with this part of the command
+    jmp .load_bin                               ; We are done with this part of the command
 
 .end_command:
     mov word [pipe_offset], 0x0000              ; Clear out our offset
-    jmp .load_file                              ; Proceed to load the file
 
-.load_file:
+.load_bin:
     mov si, program_fname                       ; Our program filename
     call conv_x3_83                             ; Convert to usable format
     jc .bad_command                             ; If error, bad command
 
+    mov byte [dir_id], 0x01                     ; Secondary directory
```

Comment:
Load the binaries from the secondary directory. For now, the secondary directory won't be changed at all.